presto-main/src/test/java/com/facebook/presto/operator/scalar/TestStringFunctions.java

```diff
@@ -334,6 +334,45 @@ public void testSubstring()
     }
 
     @Test
+    public void testVarbinarySubstring()
+    {
+        assertFunction("SUBSTR(CAST('Quadratically' AS VARBINARY), 5)", VARBINARY, new SqlVarbinary(new byte[] {'r', 'a', 't', 'i', 'c', 'a', 'l', 'l', 'y'}));
+        assertFunction("SUBSTR(CAST('Quadratically' AS VARBINARY), 50)", VARBINARY, new SqlVarbinary(new byte[] {}));
```

Comment:
You can write `VARBINARY 'Quadratically'` instead of `CAST('Quadratically' AS VARBINARY)`.